iphone/Classes/MediaModule.m

```diff
@@ -29,14 +29,11 @@
 #import <MobileCoreServices/UTCoreTypes.h>
 #import <QuartzCore/QuartzCore.h>
 #import <AVFoundation/AVFoundation.h>
-
 #import <UIKit/UIPopoverController.h>
-
-#if IS_XCODE_7_1
 #import <Photos/Photos.h>
 #import "TiUIiOSLivePhoto.h"
```

Comment:
We will set the requirements to at least Xcode 7.1 (probably 7.3 or whatever is the latest version before the 6.0.0 release).